internal/controller/controllers/kube_compat.go

```diff
@@ -39,6 +40,10 @@ func crdExists(ctx context.Context, c client.Client, crdName string) (bool, erro
 	return err == nil, client.IgnoreNotFound(err)
 }
 
+func ingressTLSSecretName(ingressName string) string {
+	return fmt.Sprintf("%s-ingress", ingressName)
```

Comment:
Where is this naming convention coming from? Not very important, but if we have control over id, should we add something to the name that would make it recognizable that's about TLS specifically? i.e. `%s-ingress-tls`

Reply:
>Where is this naming convention coming from?

Just what I happened to think of.

>should we add something to the name that would make it recognizable that's about TLS specifically? i.e. %s-ingress-tls

The secrets are created with type tls so I don't think we really need to repeat it in the name. Plus I'm not sure what other ingress-related secrets we're planning to create:

```
$ kubectl -n assisted-installer get secret
assisted-image-service-ingress   kubernetes.io/tls   3      2d23h
assisted-service-ingress         kubernetes.io/tls   3      2d23h
...
```